amd/src/main.js

```diff
@@ -0,0 +1,20 @@
+define(['jquery', 'core/ajax', 'core/templates', 'core/notification'], function ($, ajax, templates, notification) {
+    return {
+        init: function () {
+            $('#user_id').on('click', function () {
+                var promises = ajax.call([{
+                    methodname: 'block_enrol_student_enrol_student_data',
+                    args: {}
```

Comment:
Course Id needs to be a args? otherwise it will return students list for default course or other course.

Reply:
done